corehq/apps/hqwebapp/static/hqwebapp/css/proptable.css

```diff
@@ -90,3 +90,7 @@
 .case-property-name:hover i{
     display: inline-block;
 }
+
+#case-property-table tr td{
```

Comment:
hey @RynhardPietersen it's better to use `classes` rather than ids for these kind of styling additions.

I recommend using `table-case-property` as the css class. Thanks!

Reply:
looks like this could also be limited to the `case-properties-modal` id

Reply:
Thanks for the feedback! I have made the necessary changes :)